aries-site/src/examples/templates/forms/PayExample.js

```diff
@@ -81,14 +82,35 @@ const creditCardMask = [
   },
 ];
 
+const RequiredFormField = props => {
+  const { required, label, ...rest } = props;
+  return (
+    <FormField
+      label={
+        required ? (
+          <Box direction="row">
+            <Text size="xsmall">{label}</Text>
+            <Text size="xsmall">*</Text>
+          </Box>
```

Comment:
Please see if this block could be simplified with only using `label${required ? ’ *' : ''}`